data/campaigns/The_South_Guard/scenarios/01_Born_to_the_Banner.cfg

```diff
@@ -240,7 +240,7 @@
 
         [message]
             speaker=Urza Mathin
-            message= _ "Ha! This new commander is a mere boy. He will soon share the fate of that swine Loris!"
+            message= _ "Ha! This new commander is merely a boy. What kind of fool sent him here to stop us? He will perish the same way that swine Loris did!"
```

Comment:
I prefer the original "He will soon share..." but otherwise, "He will perish in the same way as that swine Loris!"